ChemistryLib/Output.cpp

```diff
@@ -7,69 +7,69 @@
  *
  */
 
-#include <fstream>
+#include <iostream>
```

Comment:
```suggestion
#include <ostream>
```

Reply:
Changed as suggested except in the `PhreeqcIO.cpp` where operator `>>` (istream) is used.